### ISSUE TITLE
[$20][Safari] Cancel and send buttons are not focus visible

### ISSUE THREAD

**Issue by kkartunov** [topcoder-platform/community-app#3083]
**Steps to Reproduce**
1. Go to https://www.topcoder.com/community/accessibility-testing 
2. click  support button 
3. hit tab key 
4. observe cancel and send button

**Expected Result**
Cancel and send button should be focused while hitting tab key

**Actual Result**
Cancel and send button are not in  focus while hitting tab key

**Device**: Desktop/Labtop
**Browser**: Safari

**WCAG Category**: 3. Understandable
**WCAG Level**: A
**WCAG Success Criterion**: 3.2.1 On Focus
**Screen Reader Used**: N/A
**Accessibility Audit Tool used**: N/A

**Attachment**
https://topcodermsft-my.sharepoint.com/personal/msft-copilots_topcodermsft_onmicrosoft_com/Documents/Forms/All.aspx?cid=09f22811%2D138e%2D4d3b%2Da8d5%2Dd3693a709128&FolderCTID=0x0120005C598A51049FC14CBA882E1AEE168F51&id=%2Fpersonal%2Fmsft%2Dcopilots%5Ftopcodermsft%5Fonmicrosoft%5Fcom%2FDocuments%2F%5BFAST%5D%20Test%20for%20Accessibility%20Series%2FCreeya%2Fbug%2D16

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097739 has been created for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by crazyk07** [comment]
Contest https://www.topcoder.com/challenges/30097739 has been updated - the new changes has been updated for this ticket.<br/><br/>```This is an automated message for crazyk via Topcoder X```

**Comment by kkartunov** [comment]
@topcoder-platform/topcodercompetitors this is open for pickup. PR against `qa-accessibility` branch.

**Comment by crazyk07** [comment]
This ticket was not processed for payment. If you would like to process it for payment, please reopen it, add the ```tcx_FixAccepted``` label, and then close it again<br/><br/>```This is an automated message for crazyk via Topcoder X```
